src/test/java/easytests/core/entities/QuizEntityTest.java

```diff
@@ -13,43 +14,24 @@
 import java.time.LocalDateTime;
 
 /**
- * @author fortyways
+ * @author miron97
  */
-public class QuizEntityTest {
+public class QuizEntityTest extends AbstractEntityTest {
+
+    private QuizzesSupport quizSupport = new QuizzesSupport();
+
     @Ignore
```

Comment:
Remove ignore, Use Model class entity, provide well configured Configuration builder (like in SubjectModelTest)